apps/analytics/analytics/sql/pv_sectors.sql

```
@@ -0,0 +1,21 @@
+WITH params AS (SELECT ?::TIMESTAMPTZ AS refreshed_at, ?::DATE AS as_of_date),
```

SQLFluff (4.3.0)

[error] Query defines CTE "params" but does not use it.

(ST03)